### ISSUE TITLE
Connection seems to be ON all the time

### ISSUE THREAD

**Issue by ssllii** [nfarina/homebridge-tesla#41]
I have been using homebridge-tesla from 2-3 years ago.  Due to other reasons, I did a complete reinstall to the latest of everything today.  It seems like the Connection to the car is always on and that seems like it would drain the battery.
Is there a way to configure to NOT keep Connection ON all the time?

**Comment by nfarina** [comment]
The plugin itself doesn’t maintain any sort of connection to the car. It only connects when Siri requests some information about the car status. It only *wakes up* the car when you try to change the state of something, like locking/unlocking. That said, I think for instance having it in Control Center might cause Siri to keep requesting car status frequently which could possibly keep the car awake depending on Tesla’s servers.

**Comment by ssllii** [comment]
Thanks for your response.
The car did finally went from "online" to "asleep" state since last night.  It seems to correspond to the time I set my phone down for the night.  
Interestingly enough, since I started using the phone, without activating anything Tesla controls from Home app, it stayed "asleep" as you described and I haven't change anything in Control Center.  I will continue to monitor to see if I can figure out what was the cause yesterday after the initial installation.
